base/src/main/java/io/spine/string/MapStringifier.java

```diff
@@ -169,7 +168,7 @@ protected String toString(Map<K, V> obj) {
             }
             return resultMap;
         } catch (Throwable e) {
-            throw newIllegalArgumentException("The exception occurred during the conversion", e);
+            throw new IllegalArgumentException("The exception occurred during the conversion", e);
```

Comment:
Let's discuss why it is changed.

Reply:
`newIllegalArgumentException(String, Object)` is tricky, as is supposes that we use it for error message formatting. Instead, here we add the `cause` Throwable to the new one.